news/2023/2023-02-06-p-community-mentorshiprogram-spring-2023-signups-now-open.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Community Mentorship Program Spring 2023 Signups Now Open"
+date: 2023-02-06 10:00:00 +0000
+---
+
+The 23rd cycle of the Community Mentorship program is on its way! Get ready to delve deep into the world of mapping, modding, or storyboarding.
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
+
+Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like contests), hang out with fellow mentees, and more!
+
+If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1716981), you'll be able to apply right away. If you're only just getting started and are eager to hone your skills, then signing up as a mentee might be what you need!
+
+---
+
+For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
+
+- Mentor signups will open on **February 6th** and close on **February 13th**.
+- Mentee signups will open on **February 27th* and close on **March 6th**.
+- The 2023 Spring cycle will run from **March 6th** to **June 5th**.
+
+Throughout this cycle, the organization team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous mapping puzzles can be found here in [this thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).
```

Comment:
is this server not public? i can't find a link to it anywhere in previous cycle announcement forum or news posts

Reply:
not for now, we have yet to do some infrastructure changes to accommodate anyone being able to join the server and possibly benefit from it. Not sure when it'll be ready